phoenicis-library/src/main/java/org/phoenicis/library/LibraryManager.java

```diff
@@ -61,19 +61,19 @@ public LibraryManager(String shortcutDirectory) {
         for (File file : directoryContent) {
             if ("shortcut".equals(FilenameUtils.getExtension(file.getName()))) {
                 ShortcutDTO shortcut = fetchShortcutDTO(shortcutDirectoryFile, file);
-                String categoryName = shortcut.getCategory();
-                if (!categoryMap.containsKey(categoryName)) {
-                    categoryMap.put(categoryName, new ArrayList<>());
+                String categoryId = shortcut.getCategory();
+                if (!categoryMap.containsKey(categoryId)) {
+                    categoryMap.put(categoryId, new ArrayList<>());
                 }
-                categoryMap.get(categoryName).add(shortcut);
+                categoryMap.get(categoryId).add(shortcut);
```

Comment:
Maybe. I didn't want to change the code at all, just fix it to work with translations.